components/kyma-environment-broker/internal/hyperscaler/account_provider.go

```diff
@@ -0,0 +1,103 @@
+package hyperscaler
+
+import (
+	"github.com/kyma-incubator/compass/components/provisioner/pkg/gqlschema"
+	"github.com/pkg/errors"
+)
+
+//go:generate mockery -name=AccountProvider
```

Comment:
please follow pattern use in different files e.g. 
```
//go:generate mockery -name=DirectorClient -output=automock -outpkg=automock -case=underscore
```
e.g. dir with mocks is called `automock`